SCClassLibrary/Common/UnitTesting/UnitTest.sc

```diff
@@ -208,16 +208,18 @@ UnitTest {
 	// waits for condition with a maxTime limit
 	// if time expires, the test is a failure
 	wait { |condition, failureMessage, maxTime = 10.0|
-		var limit = maxTime / 0.05;
+		var limit = maxTime;
+		var numberOfAttempts = 20;
```

Comment:
Before, the default behavior was to poll every 50 ms, up to 10 seconds, for 200 trials.

New behavior is to hardcode 20 trials, changing the default delta between trials to 500 ms.

The tests in this PR override maxTime, so they will respond very quickly. But will this change in default behavior affect other parts of the test suite?

I'll reserve other thoughts for the main review comment.

Reply:
ah I see, thanks!